tests/test_solidmotor.py

```diff
@@ -174,7 +188,7 @@ def tests_export_eng_asserts_exported_values_correct(solid_motor):
         "0",
         "{:2.3}".format(grain_mass),
         "{:2.3}".format(grain_mass),
-        "RocketPy"
+        "RocketPy",
```

Comment:
Should that comma exist? I believe it was created automatically, but I'm concerned if this going to create a bug or not. @Lucas-KB @lucasfourier